index.js

```diff
@@ -99,6 +99,24 @@ mOcKiNgCaSe.overrideString = () => {
   return mOcKiNgCaSe;
 };
 
+/**
+ * Overrides console.log input to print the input mOcKiNgCaSe.
+ * @param {object} [options={random: false}] Options for converting.
+ * @param {boolean} options.random=false - If case conversion should be randomized.
+ * @param {boolean} options.onlyLetters=false - If non letters characters should be removed.
+ * @param {boolean} options.firstUpper=false - If the first letter should be capitalized instead of the second when converting to mOcKiNgCaSe (e.g. MoCkInGcAsE).
+ * When combined with `options.random`, the first letter of the random string will be capitalized.
+ * @returns {object} mOcKiNgCaSe object
```

Comment:
The of **mOcKiNgCaSe** is `function` 😊

Reply:
As the TODO says and based on what I have learnt mOcKiNgCaSe that is returned is an object. Correct me if I am wrong.

Reply:
Maybe I did not pay attention and wrote a fast requirement! But you can try to check yourself! 
```js
console.log(typeof mOcKiNgCaSe);
// function
```
Sorry for the confusion!